### PR TITLE
Ncl live services

### PR DESCRIPTION
**Pull request by thirlz95**

### CODE REVIEW

File: conf/services/feedback.yml [hunk 1]
```diff
@@ -1,13 +1,56 @@
 serviceName: Feedback
-serviceHeaderName: Feedback
-serviceDescription: |
-  You can use the feedback survey to tell us how we can make our services better. We might link the feedback
-  you send us to other information about you, like your gender and age. This helps us to better understand how we
-  can improve services.
+serviceHeaderName: ''
```
Comment (Tateni93) [inline conf/services/feedback.yml:2]: Is this meant to be an empty string?

File: conf/services/income-tax.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+serviceName: Check how Much Income Tax You Paid This Year And Last Year
+serviceHeaderName: Check how much Income Tax you paid this year and last year
+serviceDescription: view your current or previous yearsâ€™ income tax calculations.
```
Comment (matthew-mcnally) [inline conf/services/income-tax.yml:3]: "yearsâ€™ income"?

File: conf/services/feedback.yml (hunk 1 above)
Comment (lewey) [inline conf/services/feedback.yml:2]: Is this not just the same as the service name?

File: conf/services/income-tax.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+serviceName: Check how Much Income Tax You Paid This Year And Last Year
+serviceHeaderName: Check how much Income Tax you paid this year and last year
```
Comment (lewey) [inline conf/services/income-tax.yml:2]: Is it OK to have different case here?

File: conf/services/income-tax.yml (hunk 2 above)
Comment (lewey) [inline conf/services/income-tax.yml:3]: odd character here??

**Review by lewey — DISMISSED** [review]

File: conf/services/feedback.yml (hunk 1 above)
Comment (thirlz95) [reply conf/services/feedback.yml:2]: added feedback, dani must have forgot to add it

File: conf/services/income-tax.yml (hunk 2 above)
Comment (thirlz95) [reply conf/services/income-tax.yml:3]: changed to years'

Comment (thirlz95) [reply conf/services/income-tax.yml:3]: changed

File: conf/services/income-tax.yml (hunk 3 above)
Comment (thirlz95) [reply conf/services/income-tax.yml:2]: git didn't pick up my change for this

**Review by Tateni93 — APPROVED** [review]

**Pull request merged**